app/components/EditorComponent.vue

```diff
@@ -22,33 +33,53 @@ interface EditorComponentProps {
   readOnly?: boolean;
 }
 
-const props: EditorComponentProps = defineProps<EditorComponentProps>();
-  const editor = new EditorJS({
-    holder: 'editorjs',
-    readOnly: Boolean(props.readOnly),
-    tools: {
-      header: {
-        class: Header as any,
-        inlineToolbar: true,
-        config: {
-          placeholder: 'Enter a header',
-          defaultLevel: 1,
-        },
-      },  
-      list: {
-        class: List as any,
-        inlineToolbar: true,
-      },
-      marker: { 
-        class: AnnoteMarker as any, // This is our custom marker tool
-        inlineToolbar: true,
+const props = defineProps<EditorComponentProps>();
+const stickyNotes = ref<Array<{ id: number; text: string; color: string }>>([]);
```

Comment:
🤔 I'm thinking for separation of concerns issues, the EditorComponent shouldn't keep track of StickyNotes, but instead be two separate things.  I don't think you need to action anything with this comment - I'm just thinking aloud.